script/vsts/release-branch-build.yml

```diff
@@ -1,6 +1,7 @@
 trigger:
 - master
 - 1.* # VSTS only supports wildcards at the end
+- electron-*
```

Comment:
👍  This will be useful in the future. We'll have to make sure we don't revert _it_ away when we revert these reversions.

Reply:
Good point. I've opened https://github.com/atom/atom/pull/18916 to revert these reversions 🥴💫😵, but this change is still intact in that pull request.